src/CMakeLists.txt

```diff
@@ -65,6 +66,7 @@ set(dream_equations
 set(dream_equation_headers
     "${PROJECT_SOURCE_DIR}/include/DREAM/Equations/Fluid/CurrentFromConductivityTerm.hpp"
     "${PROJECT_SOURCE_DIR}/include/DREAM/Equations/Fluid/AvalancheGrowthTerm.hpp"
+    "${PROJECT_SOURCE_DIR}/include/DREAM/Equations/Fluid/AvalancheGrowthTermHesslow.hpp"
```

Comment:
this file is not commited, right? 

I am thinking that we may not want to create a separate class for using the other formula, but rather giving a flag to the avalanche growth term and let that decide which Gamma to use in the calculation.

Reply:
and then also pass that flag to RunawayFluid so that only the formula that is used is evaluated.